docs/ethsimulatev1-notes.md

```diff
@@ -0,0 +1,229 @@
+							"transactionIndex": "0x0",
+							"blockHash": "0x859c932c5cf0dabf8d12eb2518e063966ac1a25e2fc49f1f02574a37f358d0b5",
+							"logIndex": "0x0",
+							"removed": false
+						}
+					],
+					"gasUsed": "0x5208",
+					"status": "0x1"
+				}
+			]
+		}
+	]
+}
+```
+
+Here the interesting part is:
+```json
+"address": "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
+"topics": [
+	"0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+	"0x000000000000000000000000c000000000000000000000000000000000000000",
+	"0x000000000000000000000000c100000000000000000000000000000000000000"
+],
+"data": "0x00000000000000000000000000000000000000000000000000000000000003e8",
+```
+In the observed event, the sender address is denoted as the `0xee...` address. The first topic (`0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef`) aligns with the event signature `Transfer(address,address,uint256)`, while the second topic (`0x000000000000000000000000c000000000000000000000000000000000000000`) corresponds to the sending address, and the third topic (`0x000000000000000000000000c100000000000000000000000000000000000000`) represents the receiving address. The quantity of ETH transacted is stored in the data field.
+
+The ETH logs will contain following types of ETH transfers:
+ - Transfering ETH from EOA
+ - Transfering ETH via contract
+ - Selfdestructing contract sending ETH
+
+But not following ones:
+ - Gas fees
+ - Multicalls eth balance override  
+
+ETH logs are not part of the calculation for logs bloom filter. Also, similar to normal logs, if the transaction sends ETH but the execution reverts, no log gets issued.
+
+## Validation
+The multicall has a feature to enable or disable validation with setting `Validation`, by default, the validation is off, and the multicall mimics `eth_call` with reduced number of checks. Validation enabled mode is intended to give as close as possible simulation of real EVM block creation, except there's no checks for transaction signatures and we also allow one to send a direct transaction from a contract.
+
+## Failures
+It is possible that user defines a transaction that cannot be included in the Ethereum block as it breaks the rules of EVM. For example, if transactions nonce is too high or low, baseFeePerGas is too low etc. In these situations the execution of multicall ends and an error is returned.
+
+## Version number
+The method name for multicall `eth_simulateV1` the intention is that after release of multicall, if new features are wanted the `eth_simulateV1` is kept as it is, and instead `eth_simulateV2` is published with the new wanted features.
+
+## Clients can set their own limits
+Clients may introduce their own limits to prevent DOS attacks using the method. We have thought of three such standard limits
+- How many blocks can be defined in `BlockStateCalls`. The suggested default for this is 256 blocks
```

Comment:
I would also argue that there is a max number, because if one allows to create only one block, then one can still request block *very high number* and this would impose a lot of phantom block hashings.